### ISSUE TITLE
Default Export Option

### ISSUE THREAD

**Issue by SilasN15** [NitzanHen/agrippa#7]
Could you introduce the option to have an `export default Navbar` at the end? I hope you understand what I mean 🙂👍

**Comment by NitzanHen** [comment]
Yes! adding a default export option for components is a good idea.

It'll be officially released in v1.2.0.
However, I already have an implementation of it that seems to work nicely. It's published as an 'unstable' release (v1.2.0-alpha.1), and you can install it with `npm install -g agrippa@next`.

(keep in mind that generally, the unstable version can be a little more buggy; right now it should be just fine, though).

What I added is an `--export-type` flag, which can receive the values `named` for named exports or `default` for default exports. So, to generate a component with a default export you can call `agrippa gen <name> --export-type default`. 
To have all components generated with default exports, you can set `exportType: "default"` in `.agripparc.json`.

Please test it out! If you run into any problems just reply and I'll try to help.

Thanks for reaching out!
